prowler/providers/common/outputs.py

```diff
@@ -111,6 +111,23 @@ def __init__(self, arguments, audit_info, mutelist_file, bulk_checks_metadata):
             self.output_filename = arguments.output_filename
 
 
+class Kubernetes_Output_Options(Provider_Output_Options):
+    def __init__(self, arguments, audit_info, mutelist_file, bulk_checks_metadata):
+        # First call Provider_Output_Options init
+        super().__init__(arguments, mutelist_file, bulk_checks_metadata)
+
+        # Check if custom output filename was input, if not, set the default
+        if (
```

Comment:
Can you include a `TODO` here to move the `output_filename` assign to the `Provider_Output_Options` class? That's because we are doing the same code in all the child classes.

Reply:
Already done